src/gui/aboutdialog.cpp

```diff
@@ -44,7 +44,9 @@ AboutDialog::AboutDialog(QWidget *parent)
     setAttribute(Qt::WA_DeleteOnClose);
 
     // Title
-    m_ui->labelName->setText(QString::fromLatin1("<b><h2>qBittorrent " QBT_VERSION " (%1-bit)</h2></b>").arg(QT_POINTER_SIZE * 8));
+    m_ui->labelName->setText(QString::fromLatin1("<b><h2>qBittorrent " QBT_VERSION "%1 (%2-bit)</h2></b>")
+                                 .arg(!strcmp(QBT_GIT_HASH, "") ? QString {} : QString::fromUtf8("+" QBT_GIT_HASH)
```

Comment:
can't you use `strlen()`?

Reply:
I would construct QString first (either empty or of the needed format) and use it here without in-place conditions.